ecpp/average.ecpp

```diff
@@ -352,12 +391,28 @@ bool database_ready;
         //escape special characters
         element_ename = std::regex_replace(element_ename, std::regex("( |\t)"), "_");
 
+        static const std::map<std::string, std::string> srcMap = {
+                                                                   {"realpower.default", "power-usage"},
```

Comment:
for readability... trim left this map definition

Reply:
done